Cubical/Algebra/DirectSum/DirectSumHIT/Properties.agda

```diff
@@ -1,20 +1,20 @@
 {-# OPTIONS --safe #-}
-module Cubical.Algebra.DirectSum.Properties where
+module Cubical.Algebra.DirectSum.DirectSumHIT.Properties where
 
 open import Cubical.Foundations.Prelude
 
 open import Cubical.Algebra.Group
 open import Cubical.Algebra.AbGroup
 
-open import Cubical.Algebra.DirectSum.Base
+open import Cubical.Algebra.DirectSum.DirectSumHIT.Base
 
 private variable
   ℓ ℓ' : Level
 
 module _ (Idx : Type ℓ) (P : Idx → Type ℓ') (AGP : (r : Idx) → AbGroupStr (P r)) where
```

Comment:
I think it is better to package the groups up, i.e. use something of type ```Idx -> AbGroup```. You can use the notation defined in 
```
Cubical.Foundations.Structures
```
to easily access the underlying type of an AbGroup with ```\< A \>```